openlcs/utils/backends.py

```diff
@@ -22,6 +22,9 @@ def configure_user(self, user):
             if node == user.username:
                 # Activate worker nodes automatically.
                 user.is_active = True
+                # Staff permission is needed for worker nodes to update
+                # component subscriptions
+                user.is_staff = True
```

Comment:
Here is only for the `openlcs-{{ env}}-worker01` users, should we add the `is_staff` for other Kerberos users that do not have superuser permission? 

Code: https://github.com/RedHatProductSecurity/openlcs/blob/667dc63a41819ebe5ad09927cf372395ce1afd53/openlcs/authentication/middleware.py#L66
such as:
```
        # Permission setting
        user.is_staff = True
        if uid in settings.OPENLCS_ADMIN_LIST:
            user.is_superuser = True
        user.save()
```

Reply:
What other users needs this permission besides admin?